testing/zim/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Adam Nye <adam@spoontech.biz>
+# Maintainer:
+pkgname=zim
+pkgver=0.66
+pkgrel=0
+pkgdesc="A Desktop Wiki Editor"
+url="http://zim-wiki.org/"
+arch="all"
+license="GPL2"
+depends="python py-gobject py-xdg py-gtk py-simplejson xdg-utils"
+makedepends=""
+install=""
+#subpackages="$pkgname-dev $pkgname-doc"
+subpackages="$pkgname-doc"
+source="http://zim-wiki.org/downloads/zim-0.66.tar.gz"
```

Comment:
All really good points, and I'll go through and make the changes you've requested now.

Its my first attempt at packaging for alpine, so still learning :)

Thanks for the advice.